ddtrace/contrib/httplib/patch.py

```diff
@@ -8,16 +8,23 @@
 from ...http import store_request_headers, store_response_headers
 from ...internal.logger import get_logger
 from ...pin import Pin
+from ...propagation.http import HTTPPropagator
 from ...settings import config
+from ...utils.formats import asbool, get_env
 from ...utils.wrappers import unwrap as _u
 
 span_name = 'httplib.request' if PY2 else 'http.client.request'
 
 log = get_logger(__name__)
 
 
+config._add('httplib', {
+    'distributed_tracing': asbool(get_env('httplib', 'distributed_tracing', default=True)),
```

Comment:
I'm not 100% about enabling this by default. We'd be adding up to 4 http headers to _every_ request which could be significant for smaller requests. It could also interfere with services not expecting these headers; or expecting a precise set headers (not sure if this is a realistic case). I think I'd rest easier with having it opt-in.

Reply:
I will change the default to False.

Reply:
```suggestion
    'distributed_tracing': asbool(get_env('httplib', 'distributed_tracing', default=False)),
```